suresh/array-str-methods.js

```diff
@@ -0,0 +1,92 @@
+var array = ['a', 1, 2, 'b', 'xyz'];
+
+function indexOf(array, ele) {
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === ele) {
+      return i;
```

Comment:
What happens if element is not in array?

Reply:
updated

Reply:
did you push up your changes?

I don't see how indexOf is handling string